SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py

```diff
@@ -2,17 +2,17 @@
 import FWCore.ParameterSet.Config as cms
 
 PixelDigitizerAlgorithmCommon = cms.PSet(
-    ElectronPerAdc = cms.double(600.0),
+    ElectronPerAdc = cms.double(1500.0),
```

Comment:
It was noticed by Slava that these numbers are repeated from https://github.com/cms-sw/cmssw/pull/31253/files#diff-4e4e386df550fa27f52ff680d9d0f183R40. Could you make them configurable form one location, if they are always intended to be the same?

Reply:
I'd rather take the clusterizer numbers from the digitizer configuration than the other way around, as for now the clusterizer is taking the simDigis in input as we lack a phase-2 unpacker/packer setup (so the clusterizer needs to stay in synch with the digitizer configuration).

Reply:
Ok that makes sense. I just pushed a new commit that has the clusterizer config import the values from the digitizer. Let me know if there is some other way people want this to be implemented.